brands/amenity/atm.json

```diff
@@ -0,0 +1,127 @@
+{
+  "amenity/atm|BMO Harris Bank~(USA)": {
+    "countryCodes": ["us"],
+    "matchNames": [
+      "BMO",
+      "BMO Bank of Montreal",
+      "BMO Harris Bank",
+      "Bank of Montreal",
+      "Harris Bank"
+    ],
+    "tags": {
+      "amenity": "atm",
+      "brand": "BMO Harris Bank",
+      "brand:wikidata": "Q4835981",
+      "brand:wikipedia": "en:BMO Harris Bank",
+      "name": "BMO Harris Bank",
+      "network": "Allpoint",
+      "network:wikidata": "Q4733264"
```

Comment:
I feel the amenity property is redundant. Can we do some JSON refactoring to avoid redundancy and to keep the common values ahead 
"amenity": "atm",

Reply:
These are redundant only because we group them into files by this property key=value pair.  The property is important as an OpenStreetMap tag, so it must stay.

Reply:
But do we have any plan to refractor this JSON/app in near future?

Reply:
No plans, I’m pretty happy with the JSON structure.